src/builder-module.c

```diff
@@ -1585,6 +1587,8 @@ builder_module_build_helper (BuilderModule  *self,
     simple = TRUE;
   else if (!strcmp (self->buildsystem, "qmake"))
     qmake = TRUE;
+  else if (!strcmp (self->buildsystem, "cargo"))
+    cargo = TRUE;
```

Comment:
this will need to be documented. See `data/flatpak-manifest.schema.json` and `doc/flatpak-manifest.xml`